### PR TITLE
fix: do not update created at on edit bounty

### LINKED ISSUE #1520: Update create date of bounty only once.
**Describe the bug**
Superadmin is displaying incorrect bounties. The query of the date range should account for bounty creation date. But instead it's showing something different. Possible latest activity. 

Here is an example. I set a custom date range from 9/9-9/15 and saw this bounty:
https://community.sphinx.chat/bounty/1302
Screenshot 2024-02-15 at 4 16 02 PM
 When I click the bounty it also shows the creation date as "a day".
Screenshot 2024-02-15 at 4 15 55 PM
But I posted this bounty 1/29
Screenshot 2024-02-15 at 4 16 16 PM

Upon further investigation, the problem is that the field "created" gets updated whenever a bounty gets saved. 

**Expected behavior**
Creation date of bounties should reflect the date it was created only. The "created" field should only get updated once, which is when a bounty gets created.

**Desktop (please complete the following information):**
 - Browser: Chrome

### Acceptance Criteria
- [ ] I have tested on Chrome desktop
- [ ] I have rebased and tested locally before submitting my PR
- [ ] I can submit a PR within 2 days of taking the bounty

Here is an [example backend test](https://github.com/stakwork/sphinx-tribes/blob/f50a099d19f22cc533b947ce04eef6c6f459fa58/handlers/organization_test.go)

### PR DESCRIPTION
**Pull request by wright-eric**
## Describe your changes
do not update created at on edit bounty

## Issue ticket number and link
closes #1520 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [x]  I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

File: frontend/app/yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,228 @@
+Arguments: 
+  /home/z0049w7n/.nvm/versions/node/v18.14.2/bin/node /home/z0049w7n/.cache/node/corepack/yarn/1.22.19/bin/yarn.js
+
+PATH: 
+  /home/z0049w7n/.nvm/versions/node/v18.14.2/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin:/home/z0049w7n/go/bin
```
Comment (elraphty) [inline frontend/app/yarn-error.log:5]: Please remove this file

File: out.csv [hunk 2]
```diff
@@ -0,0 +1,35 @@
+DatePosted,Organization,BountyAmount,Provider,Hunter,BountyTitle,BountyLink,BountyStatus,DateAssigned,DatePaid
+2024-01-07 18:12:36 +0530 IST,Guardians,1000000,Evan,Evan,Test Bounty ,https://community.sphinx.chat/bounty/1,Assigned,2024-01-07 19:10:11.410492 +0530 IST,<nil>
```
Comment (elraphty) [inline out.csv:2]: Also remove this file

**Comment by wright-eric** [comment]
@elraphty ready for review

**Comment by ecurrencyhodler** [comment]
Paid 200k sats! https://community.sphinx.chat/bounty/1533

**Pull request merged**